recipes/default.rb

```diff
@@ -87,16 +87,13 @@ class Chef::Resource
       source 'rabbitmq.upstart.conf.erb'
       owner 'root'
       group 'root'
-      mode 0644
+      mode 0o644
```

Comment:
Cookstyle autocorrect. The new version will not do this. Instead we should set this to '644'